packages/salesforcedx-vscode-core/src/context/workspaceOrgType.ts

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { Aliases, AuthInfo } from '@salesforce/core';
+import * as path from 'path';
+import * as vscode from 'vscode';
+
+import { ForceConfigGet } from '@salesforce/salesforcedx-utils-vscode/out/src/cli';
+
+export async function setupWorkspaceOrgType() {
+  const defaultUsernameOrAlias = await getDefaultUsernameOrAlias();
+  const defaultUsernameIsSet = typeof defaultUsernameOrAlias !== 'undefined';
+  let isScratchOrg = false;
+  if (defaultUsernameIsSet) {
+    const username = await getUsername(defaultUsernameOrAlias!);
+    try {
+      isScratchOrg = await isAScratchOrg(username);
+    } catch (e) {
+      if (e.name === 'NamedOrgNotFound') {
+        // If the info for a default username cannot be found,
```

Comment:
I'm guessing this will happen when the cli doesn't have any linked scratch orgs/devhub ?

Reply:
Yeah, this will happen if no local auth info file can be found for the username specified as the defaultusername.